vdev/src/app.rs

```diff
@@ -1,13 +1,28 @@
+use std::ffi::{OsStr, OsString};
 pub use std::process::Command;
-use std::{borrow::Cow, env, ffi::OsStr, path::PathBuf, process::ExitStatus, time::Duration};
+use std::{borrow::Cow, env, path::PathBuf, process::ExitStatus, time::Duration};
 
 use anyhow::{bail, Context as _, Result};
 use indicatif::{ProgressBar, ProgressStyle};
 use log::LevelFilter;
-use once_cell::sync::OnceCell;
+use once_cell::sync::{Lazy, OnceCell};
 
 use crate::config::Config;
 
+// Use the `bash` interpreter included as part of the standard `git` install for our default shell
+// if nothing is specified in the environment.
+#[cfg(windows)]
+const DEFAULT_SHELL: &str = "C:\\Program Files\\Git\\bin\\bash.EXE";
+
+// This default is not currently used on non-Windows, so this is just a placeholder for now.
```

Comment:
Do you mean because `$SHELL` will always be set? Will this still work if users are running a non-bash compatible shell like `fish`? IIRC the shebang won't be used when run as an argument to a shell executable like that 🤔

Reply:
I mean because scripts don't need to be run through a shell wrapper on non-Windows, so it's really not used.